platform/testing/end-to-end/cypress-6-upgrade-guide.md

```diff
@@ -63,6 +63,24 @@ cy.wait('@helloWorld')
   });
 ```
 
+While their functionality is similar, `cy.intercept()` does not behave exactly the same as `cy.route()`. One key difference is that previously-defined `cy.intercept()`s cannot be overriden later on in a test. For example, the second intercept defined in this test will not behave as expected because Cypress will still use the `cy.intercept()` in the `beforeEach()` block:
+
+```javascript
+beforeEach(() => {
+  cy.intercept('GET', '/todos', []) // an empty array is returned
+  cy.visit('/')
+})
+
+it('shows the initial todos', () => {
+  // this intercept will not override the original intercept
+  cy.intercept('GET', '/todos', { fixture: 'two-items.json' }) // fixture with two items returned
+  cy.visit('/')
+  cy.get('.todo').should('have.length', 2)
+})
+```
+
+If a stubbed response needs to be overridden, `cy.route()` should still be used. The Cypress development plans to release override support in a future version of Cypress. A full list of differences between `cy.route()` and `cy.intercept()` can be found [here](https://docs.cypress.io/api/commands/intercept.html#Comparison-to-cy-route).
```

Comment:
Is there a word missing here?

>The Cypress development plans to...

Was it supposed to be "The Cypress development team plans to..."?

Reply:
Good catch, it should be `development team`. Just pushed up a fix.